crates/resolver/src/error.rs

```diff
@@ -23,6 +23,7 @@ use crate::proto::{trace, ExtBacktrace};
 /// An alias for results returned by functions of this crate
 pub type ResolveResult<T> = ::std::result::Result<T, ResolveError>;
 
+#[allow(clippy::large_enum_variant)]
```

Comment:
Without them, the CI's cleanliness stage rejects this PR, because it grows the size of `Name` enough to trip over the threshold for this warning. If you're okay merging with failing CI, they could be removed.

Reply:
Ok!